lib/setupmanagers/qemuhck/network_manager.rb

```diff
@@ -156,7 +156,7 @@ def transfer_device_command(device_name, transfer_net, share_path, qemu_replacem
         net_base = "#{transfer_net}.0/24"
         smb_server = "#{transfer_net}.4"
 
-        netdev_options = "net=#{net_base},smb=#{path},smbserver=#{smb_server},restrict=on,ifname=@net_if_name@"
```

Comment:
looks like ifname parameter not supporting in User mode networking

Reply:
remove it